src/pl/senla/hotel/ie/file/EntitiesConverter.java

```diff
@@ -109,13 +71,15 @@ private String[] convertEntityToString(T t) {
         }
     }
 
-    private T convertStringToEntity(Class<? super T> clazz, String csvT) {
+    public T convertStringToEntity(Class<T> clazz, String csvT) {
```

Comment:
I don't like this huge if statements. It will grow even bigger if new entities will added to application. 
Better to create different implementations of converters for different entities and pass them into reader method to convertation or something like that.

But wait for Mark review also

Reply:
create 4 different converters again and pass them to ReaderWriters' constructors.